tests/test_core/test_stack_set.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import os
+
+from matcha_ml.core import stack_set
+from matcha_ml.config import MatchaConfigService, MatchaConfig
+from matcha_ml.errors import MatchaInputError
+
+
+def test_stack_set_valid_no_existing_file(matcha_testing_directory):
+    """Test that stack_set creates a config file if one doesn't exist and that it can be read properly.
+    Args:
+        matcha_testing_directory (str): temporary working directory
+    """
+    os.chdir(matcha_testing_directory)
```

Comment:
Nit: spacing in the code to produce more readable code (grouping logic together) - applies to the `test_stack_set_existing_file` func too